src/components/plots/areaPlot.ts

```diff
@@ -77,6 +80,59 @@ export module Plots {
       });
     }
 
+    private _generateLineDrawSteps() {
+      var drawSteps: Drawers.DrawStep[] = [];
+      if (this._dataChanged && this._animate) {
+        var attrToProjector = this._generateLineAttrToProjector();
+        var xProjector = Plot._scaledAccessor(this.x());
+        var yProjector = Plot._scaledAccessor(this.y());
+
+        var definedProjector = (d: any, i: number, dataset: Dataset) => {
+          var positionX = xProjector(d, i, dataset);
+          var positionY = yProjector(d, i, dataset);
+          return positionX != null && positionX === positionX &&
+                 positionY != null && positionY === positionY;
+        };
+
+        attrToProjector["d"] = (datum: any, index: number, dataset: Dataset) => {
+          return d3.svg.line()
+                       .x((innerDatum, innerIndex) => xProjector(innerDatum, innerIndex, dataset))
+                       .y((innerDatum, innerIndex) => this._getResetYFunction()(innerDatum, innerIndex, dataset))
+                       .defined((innerDatum, innerIndex) => definedProjector(innerDatum, innerIndex, dataset))(datum, index);
+        };
+        drawSteps.push({attrToProjector: attrToProjector, animator: this._getAnimator("reset")});
+      }
+      drawSteps.push({attrToProjector: this._generateLineAttrToProjector(), animator: this._getAnimator("main")});
+      return drawSteps;
+    }
+
+    private _generateLineAttrToProjector() {
+      var lineAttrToProjector = this._generateAttrToProjector();
+      var fillProjector = lineAttrToProjector["fill"];
+      lineAttrToProjector["stroke"] = fillProjector;
+      lineAttrToProjector["fill"] = () => "none";
+      lineAttrToProjector["vector-effect"] = () => "non-scaling-stroke";
+      lineAttrToProjector["stroke-width"] = () => "2px";
+      lineAttrToProjector["class"] = () => Line.SELECTION_CLASS;
```

Comment:
Note that the core of the problem even existing on the current implementation of `Plots.Area` is that there is no way to specify the attributes for the line and area separately.  Both are forced to infer their attributes from `_generateAttrToProjector`.

I'll play around with adding logic to the `Drawer`s (this is the only way to achieve styling) so that we can force certain ideas for attributes.